### PR TITLE
Implemented NM-57: Notify Admin

### PR DESCRIPTION
**Pull request by HomerJS**
- Add option notify admin in main menu

### CODE REVIEW

**Comment by HomerJS** [comment]
Add default data

File: app/code/local/BeC/Notify/etc/system.xml [hunk 1]
```diff
@@ -11,10 +11,31 @@
...
+                <main translate="label">
+                    <label>Main options</label>
+                    <sort_order>1</sort_order>
+                    <show_in_default>1</show_in_default>
+                    <show_in_website>1</show_in_website>
+                    <show_in_store>1</show_in_store>
+                    <fields>
+                        <admin_notify translate="label comment">
+                            <label>Notify Admin</label>
+                            <comment>Send email to admin when customer use function notify</comment>
```
Comment (aandolg) [inline app/code/local/BeC/Notify/etc/system.xml]: Why "use"? maybe "create" notify ?

File: app/locale/en_US/BeC_Notify.csv [hunk 2]
```diff
@@ -26,3 +26,9 @@
 "Delete", "Delete"
 "Are you sure?", "Are you sure?"
 "Requests were deleted!", "Requests were deleted!"
+"Notify Me", "Notify Me"
+"Main Configuration", "Main Configuration"
+"Main options", "Main options"
+"Notify Admin", "Notify Admin"
+"Send email to admin when customer use function notify", "Send email to admin when customer use function notify"
```
Comment (aandolg) [inline app/locale/en_US/BeC_Notify.csv]: Related with question about use word "use".

**Comment by HomerJS** [comment]
Change sentence

**Comment by HomerJS** [comment]
add acl

**Pull request closed without merging**